config/smtp.yml.sample

```diff
@@ -17,4 +17,10 @@ staging:
 
 # Use SendGrid for production
 production:
-  <<: *JAZZ
```

Comment:
This is all well and good, but the default configuration ISN'T currently sendgrid, so you I don't think you can sneak this in.